controllers/questioncontroller.js

```diff
@@ -48,9 +48,9 @@ router.put("/:entryId", validateSession, function (req, res) {
 
 /* ***************DELETE QUESTIONS************** */
 
-router.delete("/:entryId", validateSession, function (req, res) {
+router.delete("/delete/:id", validateSession, function (req, res) {
     
-    const query = { where: { id: req.params.entryId, owner: req.user.id } };
+    const query = { where: { id: req.params.id} };
```

Comment:
we need to add the owner ID to keep everyone from deleting questions they don't own.